src/pages/saksbehandling/lukkSøknad/Trukket.tsx

```diff
@@ -0,0 +1,105 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { FormikErrors } from 'formik';
+import { Datovelger } from 'nav-datovelger';
+import { Fareknapp, Knapp } from 'nav-frontend-knapper';
+import { Label } from 'nav-frontend-skjema';
+import { Feilmelding } from 'nav-frontend-typografi';
+import React, { SetStateAction, SyntheticEvent, useCallback, useState } from 'react';
+
+import { hentLukketSøknadBrevutkast } from '~features/saksoversikt/sak.slice';
+import { useI18n } from '~lib/hooks';
+import * as Routes from '~lib/routes';
+import { useAppDispatch, useAppSelector } from '~redux/Store';
+import { LukkSøknadType, Søknad } from '~types/Søknad';
+
+import nb from './lukkSøknad-nb';
+import styles from './lukkSøknad.module.less';
+import { LukkSøknadFormData } from './lukkSøknadUtils';
+
+const Trukket = (props: {
+    søknad: Søknad;
+    values: LukkSøknadFormData;
+    errors: FormikErrors<LukkSøknadFormData>;
+    setValues: (values: SetStateAction<LukkSøknadFormData>, shouldValidate?: boolean | undefined) => unknown;
+    handleChange: (event: React.ChangeEvent<HTMLTextAreaElement> | SyntheticEvent<EventTarget, Event>) => void;
```

Comment:
Vet ikke om jeg er så fan av at man sender inn _alt_ hit.
Hadde det gått an å sende inn kun de relevante verdiene og så å ha en `onChange` som også bare går på de verdiene som er relevante for denne komponenten?